### ISSUE TITLE
Can't share path between projects.

### ISSUE THREAD

**Issue by totalchaoslink** [DockStation/dockstation#158]
Currently our project is setup so we have different containers that can be sprung up by combining a base compose file with an override.

e.g. Setup A = docker-compose.yml + A.yml
Setup B = docker-compose.yml + B.yml

I was hoping to have each setup as it's own project but I can't create duplicate projects as they share the same path to the base docker-compose file.  I get the message "Selected path is used by other project!"

Is there a work around? Or is it possible to change this, perhaps validate on the constructed docker-compose command instead of file path to the base compose?

**Comment by warrenc5** [comment]
yeah me too. I have extensive lab setup and I want to toy with an alternate cluster configuration..

Will have a look at the SC to see what is restricting the path usage.

**Comment by warrenc5** [comment]
No code (either this project is magic or they are about to IPO) 
ok - if you're not using a toy os then create a symlink to the original lab

`ln -s lab lab-test`

Will see what breaks... :)
